Git/GitFlow.md

```diff
@@ -18,16 +18,16 @@ This image shows the git flow branches diagram:
 
 ### master
 
-Master branch should only contain a release-ready code and is one of the two branches we are releasing the code from.
+The master branch should only contain a release-ready code and is one of the two branches we are releasing the code from.
 
 ### develop
 
-Develop branch is the second branch which we release code from (for the development environment). It should only contain the code which is ready to be part of the next release. When it is decided that a new release build should be done a release branch is created from the develop branch. After a successful deployment, we merge the release branch into master and back into develop.
+Develop branch is the second branch which we release code from (for the development environment). It should only contain the code which is ready to be part of the next release. When it is decided that a new release build should be done a release branch is created from the develop branch. After a successful deployment, we merge the release branch into the master and back into the develop.
 
 ### feature
 
 Each new feature needs to be implemented in a separate branch. This is always branched off from the develop branch and then merged back into it. Feature branches also have a naming convention we should follow: **feature/feature-name**. If the project is using a feature tracking system, then it is necessary to put the ticket identifier into the name as well: **feature/T-1234-feature-name**.
 
 ### hotfix
 
-If we recognize a bug or important change request that needs to be resolved on production as soon as possible, we can create a hotfix branch. Hotfix is branched off the master branched and then merged back into master and develop branches. We use a standard hotfix naming convention which is similar to the feature one: **hotfix/T-3456-name**.
+If we recognize a bug or important change request that needs to be resolved on production as soon as possible, we can create a hotfix branch. A hotfix is branched off the master branched and then merged back into master and develop branches. We use a standard hotfix naming convention which is similar to the feature one: **hotfix/T-3456-name**.
```

Comment:
`A hotfix branch is branched off the master branched...` to `A hotfix branch is branched off the master branch...`